README.md

```diff
@@ -460,6 +464,13 @@ I would love to receive contributions to this project and welcome PRs from every
   Terminal=false
   Type=Application
   ```
+
+- __Copying images from a browser does not work correctly__  
+Depending on the clipboard utility you are using (`wl-clipboard`/`xclip` etc) the data sent to the system clipboard is read differently when copying from browser locations. 
+<br>
+If using `wayland`, Copying images from your browser should now work from most sites if using `v1.0.4` or later. This may copy the binary data as well as the metadata sting as a separate entry. Some sites/browsers may add the browser image data to the stdin in a way that `wl-clipboard` does not recognize.
```

Comment:
```suggestion
If using `wayland`, copying images from your browser should now work from most sites if using `v1.0.4` or later. This may copy the binary data as well as the metadata sting as a separate entry. Some sites/browsers may add the browser image data to the stdin in a way that `wl-clipboard` does not recognize.
```